docs/CHANGELOG.asciidoc

```diff
@@ -28,6 +28,360 @@ This section summarizes the changes in each release.
+* Adds column headers {pull}36467[#36467]
+* Adds flyout action menu with Uptime link {pull}36721[#36721]
+
+Machine Learning::
+* Adds Kibana sample dataset data recognizer modules  {pull}35138[#35138]
+* Adds File data viz filter to Index Management link {pull}36166[#36166]
+* Adds query bar to Anomaly Explorer for filtering of anomaly results for one or more influencers {pull}31763[#31763]
+* Adds the ability to add and remove filters in the Anomaly Explorer via the Anomalies table  {pull}33377[#33377]
+* Adds the ability to add and remove filters in the Anomaly Explorer via the Top influencers list {pull}33626[#33626]
+* Converts Job Selector to React {pull}35638[#35638]
+* Always show time warning message in delete modal in Job Management {pull}36729[#36729]
+* Makes layout of Edit Filter List consistent with other settings pages {pull}33421[#33421]
+* Adds created_by field to Auditbeat and APM data recognizer jobs {pull}33863[#33863]
+* Extends support for anomaly charts when model plot is enabled {pull}34079[#34079]
+* Increases the width of the text column in annotations table {pull}34450[#34450]
+* Replaces View Forecast button image with Single Metric icon {pull}34563[#34563]
+* Limits maximum annotation text length to 1000 characters {pull}34540[#34540]
+* Converts full time range selector control to EUI / React {pull}35074[#35074]
+* Uses KQL as the query language for custom URLs to Kibana pages {pull}35469[#35469]
+* Adds Metricbeat System data recognizer module {pull}35580[#35580]
+* Removes check for forecast duration 8 weeks or less {pull}35583[#35583]
+* Add links to ML jobs to the Kibana sample dataset View data menu {pull}36702[#36702]
+* Adds Data Frame UI {pull}33427[#33427]
+
+Monitoring::
+* Adds Telemetry sample data {pull}33166[#33166]
+
+Platform::
+* Introduces UI PluginsService {pull}32672[#32672]
+* Adds basic server side export API for saved objects {pull}30326[#30326]
+* Adds basic server side import API for saved objects {pull}32158[#32158]
+* Returns import errors for saved objects referencing a missing index pattern or search {pull}33005[#33005]
+* Modifies saved object export API to support the exporting of nested dependencies {pull}34225[#34225]
+* Converts the saved objects UI to use new import / export API {pull}33513[#33513]
+* Allows any type of saved object to import / export {pull}34896[#34896]
+
+Security::
+* Adds Feature Controls {pull}31652[#31652]
+* Adds OpenID Connect auth provider {pull}36201[#36201]
+* Converts account screen to React/EUI {pull}35151[#35151]
+* Exposes ability to deny ('except') access to fields via FLS {pull}26472[#26472]
+* Converts role listing page to React/EUI {pull}30970[#30970]
+
+Uptime::
+* Adds support for location name in Uptime UI {pull}36317[#36317]
+* Integrates Uptime to Logging {pull}35853[#35853]
+* Cleans up design of Uptime app {pull}31663[#31663]
+
+
+
+[float]
+[[bug-7.2.0]]
+=== Bug fixes
+
+App Architecture::
```

Comment:
Let's move these to other sections:
yml switch to Visualize
ui/field editor to Platform
IE11 to Management